redaxo/src/addons/structure/plugins/content/boot.php

```diff
@@ -10,6 +10,7 @@
 
 rex_perm::register('moveSlice[]', null, rex_perm::OPTIONS);
 rex_complex_perm::register('modules', 'rex_module_perm');
+rex_complex_perm::register('functions', 'rex_functions_perm');
```

Comment:
Vermutung: dadurch dass hier eine perm klasse registriert wird, kann man den usern/rollen jetzt via UI ein Recht "functions" einräumen, oder? Liege ich richtig?

Reply:
Es wird kein Extra-Recht angelegt, daher als complex_perm. Sonst wäre ja auch eine einfach Permission möglich gewesen.

Reply:
@staabm Die Complex-Perms erzeugen eine eigene Select-Box (mit Checkbox "Alle" darüber). Allerdings nur, wenn sie die Methode `getFieldParams()` füllen. Da es diese Complex-Perm hier nicht macht, entsteht auch nichts neues in der Rechtevergabe.

Man könnte die ganzen Rechte nun auch komplett in diese Complex-Perm verschieben, und dann also bewusst eine eigene Box dafür in der Rechtevergabe anzeigen lassen. Fände ich auch gar nicht verkehrt, muss aber nicht in diesem PR erfolgen. (Dann muss man aber auch ein entsprechenden Update-Skript erstellen).